meta/main.yml

```
@@ -1,10 +1,12 @@
 ---
 galaxy_info:
-  author: First Last
-  description: Skeleton Ansible role
+  author: Alexander King
+  description: Ansible role that downloads and installs the Metasploit Framework.
```

This wording mirrors a [similar role](https://github.com/cisagov/ansible-role-burp-suite-pro).

```suggestion
  description: Download and install the Metasploit Framework
```